src/components/learnDetail/ScriptEdit.tsx

```diff
@@ -60,107 +86,75 @@ function ScriptEdit(props: ScriptEditProps) {
       const frag = document.createDocumentFragment();
       const div = document.createElement('div');
       isLeftBlank ? (div.innerHTML = '<span class=left >/</span>') : (div.innerHTML = '<span class=right >/</span>');
+
       while (div.firstChild) {
         frag.appendChild(div.firstChild);
       }
-      range?.deleteContents();
       range?.insertNode(frag);
-      handleClickIdx(selection);
     } else if (!isOverlap && selection?.type === 'Range' && isHighlight) {
       let text = selection.toString();
 
+      if (text.includes('\n')) {
+        selection?.collapseToEnd();
+        return;
+      }
+
       if (text.includes('/')) {
         const texts = text.split('/');
```

Comment:
```suggestion
        const textList = text.split('/');
```
![image](https://user-images.githubusercontent.com/58380158/201212735-4a0fb289-00b4-43f2-b2f2-843a9000f301.png)

옛날 옛적에 이런 코드 컨벤션이 있었답니다~

Reply:
보니까 marks도 그렇던데 같이 수정합시다!

Reply:
헉스.... 감사합니다!